papis/tui/widgets/list.py

```diff
@@ -113,7 +113,7 @@ def __init__(
     def get_line_prefix(
             self,
             line: int,
-            blih: int) -> AnyFormattedText:
+            wrap_count: int = 0) -> AnyFormattedText:
```

Comment:
Why this change? The `GetLinePrefixCallable` definition does not seem to have a default value:
https://github.com/prompt-toolkit/python-prompt-toolkit/blob/d997aab538e434a6ca07d6bee226fd5b0628262f/src/prompt_toolkit/layout/controls.py#L60

Reply:
Looking there, I entirely misread the meaning of that parameter - it's so the callers can tell get_line_prefix the width of the textbox it's rendering in. The default value here was mostly to remind myself what a sane parameter to pass here is. Correcting both.